src/directory/directory.mjs

```diff
@@ -1475,7 +1465,7 @@ export const directory = {
           {
             title: 'Transcribe audio to text',
             route: '/lib-v1/predictions/transcribe',
-            filters: ['android', 'ios']
```

Comment:
Why is android taken out here?

Reply:
Not sure why Android was here in the first place. It's only supported on iOS

Reply:
I'll double-check my predictions PR when this gets merged in to make sure android is removed there as well.